pages/events.jsx

```diff
@@ -120,7 +120,9 @@ var MakerPartyExample = React.createClass({
                 <span className="span-content label-tag">Participants</span><span className="span-content">{this.props.participants}</span>
               </div> : null
             }
-            <div className="description">{this.props.description}</div>
+            <div>
+              <p className="description" dangerouslySetInnerHTML={{__html: this.props.description}}></p>
+            </div>
```

Comment:
That sounds good.

Because `description` is HTML, I think that ideally it might be good to actually call the property `descriptionHTML`, just so it's obvious to readers that it's HTML... But since we're not really following this convention anywhere else in the code yet (afaik) it's up to you if you want to make this change.